contracts/tools/DepositHelper.sol

```diff
@@ -199,6 +193,44 @@ contract DepositHelper is ReentrancyGuard {
     }
   }
 
+  function convertAndIncreaseAmount(
```

Comment:
you can send back the remaining tokens inside convertToTetuBalancerPoolToken function for removing duplicate code